eng/pipelines/libraries/build-job.yml

```diff
@@ -37,11 +44,13 @@ jobs:
           - EMSDK_PATH: $(Build.BinariesDirectory)/emsdk
         - ${{ if eq(parameters.runTests, true) }}:
           - _buildAction: -build -buildtests /p:ArchiveTests=true
-          - ${{ if ne(parameters.framework, 'allConfigurations') }}:
+          - ${{ if and(ne(parameters.framework, 'allConfigurations'), ne(parameters.useLiveCoreClr, true)) }}:
```

Comment:
Why do we want to skip restoring the tests when `useLiveCoreClr` is not being used?

Reply:
I'll double-check this, I may have confused this with the other property to not restore the CoreCLR framework (as we're using the live one).

Reply:
Yeah. That property is to avoid restoring libraries test-projects. AllConfigurations is a special framework which doesn't build libraries test projects at all (it builds custom package tests) so we don't need to restore test projects as part of its restore step.